wkcuber/api/Properties/DatasetProperties.py

```diff
@@ -134,7 +140,12 @@ def _from_json(cls, path: Union[str, Path]) -> "WKProperties":
                     )
 
             return cls(
-                path, data["id"]["name"], data["scale"], data["id"]["team"], data_layers
+                path,
+                data["id"]["name"],
+                data["scale"],
+                data["id"]["team"],
+                data_layers,
+                data.get("defaultViewConfiguration"),
```

Comment:
Thanks for noticing the missing export. Since these values are webKnossos specific and only `wkw` can be read by webKnossos, I think it's fine to not support these values for `TiffDatasets`